aspnetcore/includes/webapp-alias-notice.md

```diff
@@ -0,0 +1,2 @@
+> [!NOTE]
+> With the release of ASP.NET Core 2.1, `webapp` is an alias of the `razor` Razor Pages template name when creating a project with the [dotnet new](/dotnet/core/tools/dotnet-new) command. If executing `dotnet new webapp <OPTIONS>` doesn't result in the generation of a new Razor Pages app, install the [.NET Core 2.1 SDK](https://www.microsoft.com/net/download/dotnet-core/sdk-2.1.300).
```

Comment:
If the `dotnet new webapp <OPTIONS>` command generates an error, verify  [.NET Core 2.1 SDK](https://www.microsoft.com/net/download/dotnet-core/sdk-2.1.300) is installed.

Reply:
In  ASP.NET Core 2.1 and later, `webapp` is an alias of the `razor` as an argument in the [dotnet new](/dotnet/core/tools/dotnet-new) command.

Reply:
Something like that.